apps/explorer/lib/explorer/chain/import/runner/celo_accounts.ex

```diff
@@ -62,14 +62,26 @@ defmodule Explorer.Chain.Import.Runner.CeloAccounts do
     {:ok, accounts}
   end
 
+  defp handle_dedup([head | lst]) do
```

Comment:
Do you need to deconstruct into `head | lst` here? I though `Enum.reduce` does this for you?